Markup Sign_in ＆ Sign_up

isoppu1632 opened the pull request:
#WHAT
・Sign_upの「登録方法選択」、「会員情報入力」、「電話番号承認」、「お届け先住所入力」、「支払方法」、「完了」のマークアップ
・Sign_inのログイン画面
・選択プルの項目ファイル

#WHY
・ユーザー登録の為

Commit:
first commit

Commit:
Merge branch 'master' into Markup_Sign_up

Commit:
Merge branch 'master' into Markup_Sign_up

Commit:
Merge branch 'master' into Markup_Sign_up

Commit:
Merge branch 'master' into Markup_Sign_up

Commit:
Merge branch 'master' into Markup_Sign_up

Commit:
rails erb → haml

rails erbをhamlに変換

Commit:
Create sign_up & sign_in

#WHAT
・Sign_upの「登録方法選択」、「会員情報入力」、「電話番号承認」、「お届け先住所入力」、「支払方法」、「完了」のマークアップ
・Sign_inのログイン画面
・選択プルの項目ファイル

#WHY
・ユーザー登録の為

yuki-uchida commented on app/assets/stylesheets/_test_user.scss:
```diff
@@ -0,0 +1,677 @@
+}
+
+//header__nav-barの線の部分
+.progress-status{
```
BEMに従って、親要素の名前を継承したクラス命名をしてください

yuki-uchida commented on app/assets/stylesheets/_test_user.scss:
```diff
@@ -0,0 +1,677 @@
+}
+
+//誕生日の選択プル
+.birthday {
```
BEMを用いたCSSでは、様々な場所で使われるCSSはクラスを複数付与するのではなく、mixinで呼び出す形で記述します。

yuki-uchida commented on app/views/devise/registrations/select/_prefecture_id.haml:
```diff
@@ -0,0 +1,51 @@
+.prefecture_id
+  .select-wrap.select-default
+    %select{:name => "prefecture_id"}
+      %option{:value => "1"} 北海道
```
active_hashを使ってください

yuki-uchida reviewed (changes requested):
CSSは全体的に見直してください

Commit:
Fix Sign_in & Sign_up

#WHAT
・SassのMixinの使用
・BEMの見直し
・active_hashを使い為、余分なselectディレクトリを削除

#WHY
- BEMに従って、親要素の名前を継承
- クラスを付与するのではなくmixinを活用
- 日付や都道府県はactive_hashを使う為、省略

hiroki5123 commented:
変更差分が大きすぎるので次からもう少し小さい単位でプルリクエストを作ってください。

hiroki5123 commented:
また、ビューファイルを作った時はこちらに画面のキャプチャを添付してください。

hiroki5123 commented on app/views/layouts/application.html.haml:
```diff
@@ -0,0 +1,12 @@
+    = csp_meta_tag
+    = stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload'
+    = javascript_include_tag 'application', 'data-turbolinks-track': 'reload'
+    %link{:href => "https://use.fontawesome.com/releases/v5.6.3/css/all.css", :rel => "stylesheet"}/
```
ハッシュロケット使わないようにしましょう

hiroki5123 replied:
font-awesomeのgem入れているみたいなので、こちら削除してください。

Commit:
Fix Sign_up & Sign_in

#WHAT
・footerの部分テンプレート化
・余計な記述の削除

#WHY
・同じ記述が多かった為
・不要な記述があった

isoppu1632 commented:
![screencapture-localhost-3000-users-sign_in-2019-04-14-20_56_26](https://user-images.githubusercontent.com/48373985/56092574-c3b79700-5ef8-11e9-9dc9-9f7e02e6e467.png)
![screencapture-localhost-3000-users-sign_up-2019-04-14-20_49_57](https://user-images.githubusercontent.com/48373985/56092575-c3b79700-5ef8-11e9-8374-335189a16dfe.png)
![screencapture-localhost-3000-users-sign_up-2019-04-14-20_51_14](https://user-images.githubusercontent.com/48373985/56092576-c3b79700-5ef8-11e9-8ef4-02732f569c67.png)
![screencapture-localhost-3000-users-sign_up-2019-04-14-20_51_45](https://user-images.githubusercontent.com/48373985/56092577-c3b79700-5ef8-11e9-947f-f16ea5e83273.png)
![screencapture-localhost-3000-users-sign_up-2019-04-14-20_52_06](https://user-images.githubusercontent.com/48373985/56092578-c4502d80-5ef8-11e9-9255-91570d9713c6.png)
![screencapture-localhost-3000-users-sign_up-2019-04-14-20_52_31](https://user-images.githubusercontent.com/48373985/56092579-c4502d80-5ef8-11e9-8414-6d449d96c78d.png)
![screencapture-localhost-3000-users-sign_up-2019-04-14-20_55_22](https://user-images.githubusercontent.com/48373985/56092580-c4502d80-5ef8-11e9-96a3-3844680f827d.png)

hiroki5123 commented on app/assets/stylesheets/application.scss:
```diff
@@ -1,7 +1,7 @@
 @import "./colors";
 @import "./mixin";
-@import "./items";
+// @import "./items";
```
使っていないのであれば削除しましょう

isoppu1632 replied:
トップページを担当していた方のCSS部分です。
自分の作っている所に干渉して崩れてしまっている所があったので
一時的にコメントアウトにしています。
担当者が直し次第戻す予定です。

hiroki5123 replied:
承知しました。

hiroki5123 commented:
指摘された部分に関して修正を行わない理由を教えてください。

hiroki5123 commented on app/views/devise/confirmations/new.html.haml:
```diff
@@ -0,0 +1,10 @@
+%h2 Resend confirmation instructions
```
こちらは現時点で使う予定があるファイルですか？
もしないのであれば削除しましょう。
他のファイルも現時点で使う予定のないファイルが変更差分に入っていれば削除してください。

isoppu1632 replied:
余分なDeviseのviewファイルを削除しました。

Commit:
Organize_devise_file

#WHAT
・deviseのviewの不要なファイルを削除

#WHY
・必要のないファイルは整理

miyagit commented on app/views/devise/registrations/_new_credit.haml:
```diff
@@ -0,0 +1,56 @@
+          .expiration
+            .select-wrap.select-wrap__month.select-default
+              %select{:name => "month"}
+                %option{:value => ""} --
```
ハッシュロケットを直しましょう。

isoppu1632 replied:
修正いたしました。

miyagit reviewed (commented):
ハッシュロケットを使ってるところが蹴っこうあるので見直しましょう。

Commit:
Fix Hashrocket

#WHAT
・ハッシュロケットを使用しない

#WHY
・古い書き方

kuroko918 commented:
いいかと思います！
LGTMです！
